src/Components/Stats/Stats.module.scss

```diff
@@ -18,13 +19,170 @@
 }
 
 .line {
-  @include line()
+  @include line();
+  margin: auto;
+  margin-top: 0.3em;
+  margin-bottom: 0.3em;
 }
```

Comment:
Glad to see this lil biddy Line get used throughout the project! It honestly might make sense to turn this lil line div into a component at some point, seeing as we've been using it kinda frequently.